crates/pathfinder/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
+authors = [
+    "Mirko von Leipzig <mirko.vonleipzig@gmail.com>",
+    "Krzysztof Lis <klis33@gmail.com>",
+    "Joonas Joonas Koivunen <joonas@equilibrium.co>"
+]
+name = "pathfinder"
+version = "0.1.0"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+name = "pathfinder_lib"
+path = "src/lib.rs"
+
+[dependencies]
+anyhow = "1.0.44"
+bigdecimal = { version = "0.3.0", features = ["serde"] }
+# paritys scale codec locks us here
+bitvec = "0.20.4"
+clap = "2.33.3"
+enum-iterator = "0.7.0"
+# Use our fork of ff. ff currently causes a bitvec-funty clash with our ethereum deps.
+# We've modified ff to not rely on bitvec (note that ff features are broken: https://github.com/zkcrypto/ff/issues/69).
+# We can use normal ff once it and `ethabi` (via `web3`) play nice.
+ff = { git = "https://github.com/eqlabs/ff", branch = "derive_bitvec", default-features = false, features = [
+    "derive",
+    "alloc",
+] }
```

Comment:
```suggestion
```

Reply:
That does make sense to me. Do you have a toml formatter extension that you use?